dashboard/app/mailers/pd/workshop_mailer.rb

```diff
@@ -98,13 +106,22 @@ def teacher_enrollment_reminder(enrollment, days_before: nil)
     @pre_workshop_survey_url = enrollment.pre_workshop_survey_url
     @is_first_pre_survey_email = days_before == INITIAL_PRE_SURVEY_DAYS_BEFORE
 
+    from = from_teacher
```

Comment:
Can we put this first part in an `else` so that we treat these as immutable values rather than overwriting them immediately in some cases?

Reply:
Done.